From 3071757198f9272ee4028c7bff4eacaf1efd62b8 Mon Sep 17 00:00:00 2001
Message-Id: <3071757198f9272ee4028c7bff4eacaf1efd62b8.1368111913.git.minovotn@redhat.com>
In-Reply-To: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
References: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 24 Apr 2013 08:17:38 +0200
Subject: [PATCH 04/65] Revert "virtio-console: Enable port throttling when
 chardev is slow to consume data"

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <a677ff1e97a67d78344d8fe1b4ca383670ccc0a5.1366724981.git.amit.shah@redhat.com>
Patchwork-id: 50782
O-Subject: [RHEL6.5 qemu-kvm PATCH 04/65] Revert "virtio-console: Enable port throttling when chardev is slow to consume data"
Bugzilla: 909059
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

This reverts commit b3d706dd43977c6be2eb5fe042d3cf9c21914963.
---
 hw/virtio-console.c | 11 -----------
 1 file changed, 11 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/virtio-console.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/hw/virtio-console.c b/hw/virtio-console.c
index a2e9bf0..434c160 100644
--- a/hw/virtio-console.c
+++ b/hw/virtio-console.c
@@ -19,16 +19,6 @@ typedef struct VirtConsole {
     CharDriverState *chr;
 } VirtConsole;
 
-/*
- * Callback function that's called from chardevs when backend becomes
- * writable.
- */
-static void chr_write_unblocked(void *opaque)
-{
-    VirtConsole *vcon = opaque;
-
-    virtio_serial_throttle_port(&vcon->port, false);
-}
 
 /* Callback function that's called when the guest sends us data */
 static ssize_t flush_buf(VirtIOSerialPort *port, const uint8_t *buf, size_t len)
@@ -114,7 +104,6 @@ static const QemuChrHandlers chr_handlers = {
     .fd_can_read = chr_can_read,
     .fd_read = chr_read,
     .fd_event = chr_event,
-    .fd_write_unblocked = chr_write_unblocked,
 };
 
 static const QemuChrHandlers chr_handlers_no_flow_control = {
-- 
1.7.11.7

